eng/Versions.props

```diff
@@ -24,7 +24,7 @@
     <MicrosoftCodeAnalysisTestingVersion>1.1.2-beta1.23411.1</MicrosoftCodeAnalysisTestingVersion>
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.149-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
-    <CodeStyleAnalyzerVersion>4.6.0</CodeStyleAnalyzerVersion>
+    <CodeStyleAnalyzerVersion>4.8.0-3.final</CodeStyleAnalyzerVersion>
```

Comment:
><CodeStyleAnalyzerVersion>4.8.0-3.final</CodeStyleAnalyzerVersion> [](http://example.com/codeflow?start=4&length=66)

Is the comment in the line above no longer applicable?

Reply:
I couldn’t find a newer version of dotnet-format than the one we currently reference.